src/li_wang_wu.py

```
@@ -0,0 +1,48 @@
+from utils import *
```

Ruff (0.14.8)

`from utils import *` used; unable to detect undefined names

(F403)